_15thnight/core.py

```diff
@@ -130,3 +131,26 @@ def send_out_resolution(need):
                 body=body
             )
         )
+
+    # Check if alert is closed, if so, send out resolution notices
+    send_alert_resolution_notice.apply_async(args=[need])
```

Comment:
Oh, snap. This couldn't have been ran that way. send_alert_resolution_notice isn't a celery task. I'm still not a fan of such a large method and it needs to be separate for testing anyway.